src/VolumeRenderSettings.ts

```diff
@@ -214,21 +203,21 @@ export class VolumeRenderSettingUtils {
   }
 
   /**
-   * Creates a deep copy of a VolumeRenderSettings object.
+   * Creates a deep copy of this VolumeRenderSettings object.
    * @param src The object to create a clone of.
    * @returns a new VolumeRenderSettings object with identical fields that do not
    * share references with the original settings object.
    */
-  public static clone(src: VolumeRenderSettings): VolumeRenderSettings {
-    const dst = defaultVolumeRenderSettings(new Volume()); // initialize with empty volume
-    for (const key of Object.keys(src)) {
-      const val = src[key];
+  public clone(): VolumeRenderSettings {
+    const dst = new VolumeRenderSettings(new Volume()); // initialize with empty volume
```

Comment:
maybe the arg to the ctor could be optional so you don't have to construct a new Volume here?